react/Button/Readme.md

```diff
@@ -81,7 +81,7 @@ const { Button } = require('./index');
 const Chip = require('../Chip').default;
 
 <div>
-  <p><Button extension="full" size="large" label='Label' extraRight={<Chip size="small" theme="primary" className="u-m-0">1/2</Chip>} /></p>
+  <p><Button theme="ghost" extension="full" size="large" label='Label' extraRight={<Chip size="tiny" theme="full" className="u-m-0">1/2</Chip>} /></p>
```

Comment:
I don't think this should be part of the diff?

Reply:
Mmmmh Ok I'll make another PR out of it.

Reply:
Well if it's not a mistake and actually needs to land on master, I don't mind leaving it as part of this PR :)

Reply:
It wasn't a mistake but you're kinda right anyway, it was intended to update the `extraRight` demo with the new Chip theme & size but it's not really related to the core of this PR. 

I already removed it so I'll add it later in a clean & independant PR once this once is merged.